06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,142 @@
+const invertedIndex = new InvertedIndex();
+
+
+const Book = require('./book.json');
```

Comment:
A base filename should exactly match the name of its default export [See link](https://github.com/airbnb/javascript#naming--filename-matches-export)